jcl/src/java.base/share/classes/java/lang/StringBuffer.java

```diff
@@ -803,8 +803,8 @@ public synchronized char charAt(int index) {
  * @param		end	the offset one past the last character
  * @return		this StringBuffer
  *
- * @exception	StringIndexOutOfBoundsException when <code>start < 0, start > end</code> or
- *				<code>end > length()</code>
+ * @exception	StringIndexOutOfBoundsException
+ *				If {@code start} is negative, {@code start} is grater {@code end} or {@code end} is grater length()
```

Comment:
grater - greater than (two places)

Reply:
Some how I missed this line.. actually this should be fixed with this line,
when {@code}